test/client-request.js

```diff
@@ -370,3 +370,27 @@ test('request post body no extra data handler', { skip: nodeMajor < 16 }, (t) =>
     t.pass()
   })
 })
+
+test('request with onInfo callback', (t) => {
+  t.plan(2)
+  const infos = []
+  const server = createServer((req, res) => {
+    res.writeContinue()
+    res.setHeader('Content-Type', 'application/json')
+    res.end(JSON.stringify({ foo: 'bar' }))
```

Comment:
Can you add a test where the response is ended after the client receives onInfo?

Reply:
Sure